packages/portal/src/components/landing/LandingEmbed.vue

```diff
@@ -64,12 +62,6 @@
         type: Object,
         default: null
       }
-    },
-
-    data() {
-      return {
-        containerId: kebabCase(this.englishTitle)
```

Comment:
`englishTitle` prop is not used anywhere anymore, so can be removed

Reply:
Removed.